### PR TITLE
Feature/filter problem

### PR DESCRIPTION
**Pull request by kimcodingvv**

### CODE REVIEW

File: JudgeWeb/backend/models/schema.model.js [hunk 1]
```diff
@@ -71,7 +78,7 @@ const testcaseSchema = new mongoose.Schema({
     },
     out: {
         txt: String
-    }
+    },
```
Comment (kyw04004) [inline JudgeWeb/backend/models/schema.model.js]: 쉼표 불편

**Review by kyw04004 — COMMENTED** [review]
Good

File: JudgeWeb/backend/models/index.model.js [hunk 2]
```diff
@@ -186,6 +208,26 @@ export class ProblemModel {
             throw new Error('Model -> deleteTestcase error');
         }
     }
+    static updateOpenTime = async (problems, openTime) => {
+        logger.info(problems);
```
Comment (Hwangsoomin) [inline JudgeWeb/backend/models/index.model.js:215]: delete logger

Comment (Hwangsoomin) [inline JudgeWeb/backend/models/index.model.js]: delete logger

File: JudgeWeb/frontend/src/views/problemList.vue [hunk 3]
```diff
@@ -35,22 +35,39 @@ export default {
...
+                    (response) => {
+                        this.result = response.data;
+                    },
+                    (error) => {
+                        this.$log.info(3);
+                        alert('에러1' + error.response.data.error);
+                    }
+                )
+                .catch(error => {
+                    this.$log.info(4);
```
Comment (Hwangsoomin) [inline JudgeWeb/frontend/src/views/problemList.vue]: delete 에러1, 에러2  log, alert

Comment (kimcodingvv) [reply JudgeWeb/frontend/src/views/problemList.vue]: 다음 수정 때 반영

**Review by Hwangsoomin — APPROVED** [review]
OK

**Pull request merged**